services/validatorWorker/lib/followerRules.js

```diff
@@ -47,4 +47,19 @@ function sumMins(our, approved) {
 	)
 }
 
-module.exports = { isValidTransition, isHealthy }
+function getRootHash(channel, balances, adapter){
```

Comment:
I think you can export that and reuse it in `leader.js`, right?

Reply:
yeah definitely